README.md

```diff
@@ -97,7 +97,7 @@ app
 
 #### localeDirs
 
-Type: `Array`  
```

Comment:
markdown requires trailing space to insert a newline.

You've just changed the formatting to make these two lines one.  No good.